pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -435,8 +435,13 @@ type ZoneSpec struct {
 type CephObjectStoreUser struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata"`
-	Spec              ObjectStoreUserSpec `json:"spec"`
-	Status            *Status             `json:"status"`
+	Spec              ObjectStoreUserSpec    `json:"spec"`
+	Status            *ObjectStoreUserStatus `json:"status"`
+}
+
+type ObjectStoreUserStatus struct {
+	Phase      string `json:"phase,omitempty"`
+	SecretName string `json:"secretname"`
```

Comment:
What about adding an `Info` field instead, which will be a map of strings that we can populate with any values. See https://github.com/rook/rook/pull/5774 for reference.

Reply:
> What about adding an `Info` field instead, which will be a map of strings that we can populate with any values. See #5774 for reference.

Done.